spec/formatting.md

```diff
@@ -361,7 +361,7 @@ An _expression_ fails to resolve when:
 
 The _fallback value_ depends on the contents of the _expression_:
 
-- _expression_ with _literal_ _operand_ (_quoted_ or _unquoted_):
+- _expression_ with any _literal_ _operand_:
```

Comment:
This one is tricky, since it eliminates what I thought was a good reminder about literals.

```suggestion
- _expression_ with a _literal_ _operand_ (either quoted or unquoted)
```